Installing-and-Configuring/OS/Qubes.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: default
+title: Qubes
+permalink: /OS/Qubes
+nav_order: 1
+parent: Operating Systems
+grand_parent: Installing and configuring
+has_toc: true
+---
+
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Installing Qubes
+===
+
+![Heads splash screen]({{ site.baseurl }}/images/Heads_splash_screen.jpg)
+
+Plus in the USB stick with the R4.0 install media into one of the USB port and
+ boot into USB mode (hit 'u' at the prompt), then boot using this option:
+
+```text
+2. Install Qubes R4.0 [kernel /isolinux/xen.gz console=none]
+```
+
+If that completes with no errors it will launch the Xen hypervisor from the
+ ROM image and start the Qubes installer.  The first few seconds are run
+ with an archaic video mode, so things appear a little weird, but once the dom0
+ kernel initializes the graphics it should look right.
+
+![Qubes partitioning]({{ site.baseurl }}/images/Qubes_partitioning.jpg)
+
+Use default QubesOS partitioning scheme for QubesOS 4.x
+
+![Disk Recovery Key]({{ site.baseurl }}/images/Disk_encryption_recovery_key.jpg)
+
+The Disk Recovery Key that you enter here will be used as the
+"recovery password" later.  It should be a long value since you won't
+have to enter it very often; only when upgrading the Heads firmware
+or if there is a need to recover the disk on an external machine.
+You will need it again shortly, so don't lose it yet.
```

Comment:
You should never loose the Disk recovery key. This key passphrase will need to be reentered when setting https://osresearch.net/Keys/#tpm-disk-encryption-key

This option is offered from the GUI (again lets not forget that going into recovery invalidates PCR measurements, and that having kernel modules loaded mismatch between the moment of setting the TPM disk encryption key will not fly. This is why this should be done from the GUI by saving a new boot default option and answering Y to `Do you wish to add a disk encryption to the TPM [y/N]: `

Reply:
incorporated